ckanext/validation/utils.py

```diff
@@ -17,6 +18,8 @@
 import ckan.lib.uploader as uploader
 from ckan import model
 
+from ckanext.s3filestore.uploader import S3ResourceUploader
```

Comment:
This shouldn't be unconditionally imported, or it will break any context that doesn't use ckanext-s3filestore, such as our CI build...